Artifacts/linux-deprovision/depro.sh

```diff
@@ -2,7 +2,7 @@
 
 main () {
     { # try
-        echo 'waagent -force -deprovision+user && poweroff' | at now + 1 minute > /dev/null 2>&1
+        echo 'waagent -force -deprovision+user' | at now + 1 minute > /dev/null 2>&1
```

Comment:
But you already have a 2 minutes delay in the postDeployActions. Do you need both?